src/Moq/Obsolete/MockFactory.cs

```diff
@@ -293,6 +293,30 @@ public Mock<T> Create<T>(MockBehavior behavior, params object[] args)
 			return CreateMock<T>(behavior, args);
 		}
 
+		/// <summary>
+		/// Creates an instance of the mock using the given constructor call including its
+		/// argument values and with a specific <see cref="MockBehavior"/> behavior.
+		/// </summary>
+		/// <typeparam name="T">Type to mock.</typeparam>
+		/// <param name="newExpression">Lambda expression that creates an instance of <typeparamref name="T"/> T.</param>
```

Comment:
Same as in the other review comment, there's a superfluous `T` after the `<typeparamref/>`.